app/scripts/modules/core/src/modal/wizard/WizardModal.tsx

```diff
@@ -180,11 +182,12 @@ export class WizardModal<T = {}> extends React.Component<IWizardModalProps<T>, I
                   spinner
                 ) : (
                   <div className="row">
-                    <div className="col-md-3 hidden-sm hidden-xs">
-                      <ul className="steps-indicator wizard-navigation">{pageLabels}</ul>
-                    </div>
-
-                    <div className="col-md-9 col-sm-12">
+                    {!hideWizardNavigation && (
+                      <div className="col-md-3 hidden-sm hidden-xs">
+                        <ul className="steps-indicator wizard-navigation">{pageLabels}</ul>
+                      </div>
+                    )}
+                    <div className={!hideWizardNavigation ? 'col-md-9 col-sm-12' : 'col-md-12 col-sm-12'}>
```

Comment:
This *might* read a little clearer either using `classNames`?

Reply:
Or reversing the ternary condition and maybe explicitly default-prop-ing `hideWizardNavigation` to false? Or even changing the prop name to `includeWizardNavigation` and defaulting to true? Or ignore me, I'm v. bad at intuiting ternaries :rofl:

Reply:
lol I reversed it. I hope it makes it easier to read now